src/libs/actions/Card.ts

```diff
@@ -189,5 +189,25 @@ function setIssueNewCardStep(step: IssueNewCardStep | null) {
     Onyx.merge(ONYXKEYS.ISSUE_NEW_EXPENSIFY_CARD, {currentStep: step});
 }
 
-export {requestReplacementExpensifyCard, activatePhysicalExpensifyCard, clearCardListErrors, reportVirtualExpensifyCardFraud, revealVirtualCardDetails, setIssueNewCardStep};
+function setIssueNewCardDataAndGoToStep(data: Partial<IssueNewCardData>, step: IssueNewCardStep) {
```

Comment:
@koko57 Currently, we have 3 functions setIssueNewCardStep, setIssueNewCardDataAndGoToStep and clearIssueNewCardFlow

It works well. But I think we can minimalize it into one function
```
function setIssueNewCardDataAndGoToStep(data?: Partial<IssueNewCardData>, currentStep?: IssueNewCardStep) {
    if (currentStep) {
        Onyx.merge(ONYXKEYS.ISSUE_NEW_EXPENSIFY_CARD, {data, currentStep}) 
        return
    } 
    Onyx.set(ONYXKEYS.ISSUE_NEW_EXPENSIFY_CARD, {
        currentStep: null,
        data: {},
    })
}

```

Reply:
I would rather have the function with `merge` and `set` separate, but I can merge these  two:  setIssueNewCardStep, setIssueNewCardDataAndGoToStep

Reply:
Great :3